src/http/common.cc

```diff
@@ -119,7 +119,9 @@ class http_chunked_data_sink_impl : public data_sink_impl {
 public:
     http_chunked_data_sink_impl(output_stream<char>& out) : _out(out) {
     }
-    virtual future<> put(net::packet data)  override { abort(); }
+    virtual future<> put(net::packet data) override {
```

Comment:
The http_content_length_data_sink_impl deserves the same update

Reply:
Done.

I am unfamiliar with this data sink; would you know a useful way to include a unit test for this one?

Reply:
Currently only client instantiates one. Check test_basic_content() helper for the example of creating and using a client

Reply:
Sorry for the delay, and thank you for your suggestion, I've now added a unit test for it via `test_basic_content()`.